frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/review-tab.tsx

```diff
@@ -12,11 +12,32 @@ export const ReviewTab: React.FC<ReviewTabProps> = ({ wizardReduxID }) => {
       <Title headingLevel="h3" size="lg" className="kubevirt-create-vm-modal__review-tab-title">
         Review and confirm your settings
       </Title>
-      <VMSettingsTab wizardReduxID={wizardReduxID} isReview />
+      <Title
+        headingLevel="h4"
+        size="lg"
+        className="kubevirt-create-vm-modal__review-tab-lower-section"
+      >
+        General
+      </Title>
+      <GeneralReview wizardReduxID={wizardReduxID} />
+      <Title
+        headingLevel="h4"
+        size="lg"
+        className="kubevirt-create-vm-modal__review-tab-lower-section"
+      >
+        Networking
+      </Title>
       <NetworkingReview
         wizardReduxID={wizardReduxID}
         className="kubevirt-create-vm-modal__review-tab-lower-section"
       />
+      <Title
+        headingLevel="h4"
+        size="lg"
+        className="kubevirt-create-vm-modal__review-tab-lower-section"
+      >
+        Storage
+      </Title>
       <StorageReview
         wizardReduxID={wizardReduxID}
         className="kubevirt-create-vm-modal__review-tab-lower-section"
```

Comment:
Can you add the advanced section as well? Maybe we could hide it if not touched?

The boot order section would be nice as well to have

Reply:
Ah sorry, I missed the TODOs ^^